composer.json

```diff
@@ -56,6 +56,7 @@
     "conflict": {
         "friendsofsymfony/rest-bundle": "<1.7.4 || >=3.0",
         "jms/serializer": "<0.13",
+        "liip/imagine-bundle": "<1.0",
```

Comment:
Is there a reason why we test version `^1.9`, but only conflict versions lower than 1.0? Shouln't they match when running tests?

Reply:
Hmm, correct. Should I remove this line entirely?

Reply:
No, just do <1.9

Reply:
@core23  can you please review your requested changes?